components/plan-build/bin/hab-plan-build.sh

```diff
@@ -2536,9 +2587,14 @@ _resolve_dependencies
 
 _set_path
 
-# Download the source
+# Ensure the cache folder is created.
 mkdir -pv "$HAB_CACHE_SRC_PATH"
-do_download
+
+# Set the download or copy command variable.
+_set_get_source_cmd
+
+# Download or copy the source.
+$($GET_SOURCE_CMD)
```

Comment:
Maybe changing this to `SOURCE_FETCH_CMD` or something might be clearer. I am open to suggestions.